src/main/java/tira/Tira.java

```diff
@@ -0,0 +1,82 @@
+package tira;
+
+import tira.task.TaskList;
+
+import java.util.Scanner;
+import java.io.InputStreamReader;
+import java.io.PrintWriter;
+import java.util.ArrayList;
+import java.util.List;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.io.FileReader;
+import java.io.FileNotFoundException;
+import java.io.File;
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+import java.time.format.DateTimeParseException;
+
+/**
+ * Tira java
+ *
+ */
+
+public class Tira {
+    private static final String Directory = "./data";
+    private static final String FileName = "./data/Tira.txt";
```

Comment:
It might be good to use SCREAMING_SNAKE_CASE for these constant names instead.
```suggestion
    private static final String DIRECTORY = "./data";
    private static final String FILENAME = "./data/Tira.txt";
```